variables.tf

```diff
@@ -2,8 +2,60 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "resource_group_id" {
+  description = "The Id of an existing IBM Cloud resource group where the watsonx Assistant instance will be grouped."
+  type        = string
+}
+
+variable "region" {
+  description = "IBM Cloud region where watsonx Assistant instance will be created."
+  type        = string
+}
+
+variable "tags" {
+  description = "Metadata labels describing this watsonx Assistant instance."
+  type        = list(string)
+  default     = []
+}
+
+variable "watsonx_assistant_name" {
+  description = "The name of the watsonx assistant instance."
+  type        = string
+}
+
+variable "existing_assistant_instance" {
+  description = "CRN of the an existing watsonx Assistant instance."
+  type        = string
+  default     = null
+}
+
+variable "watsonx_assistant_plan" {
+  description = "The plan that is required to provision the watsonx Assistant instance."
+  type        = string
+  default     = "trial"
+  validation {
+    condition = anytrue([
+      var.watsonx_assistant_plan == "do not install",
```

Comment:
there is no need to support `do not install`

Reply:
Sure, I will revert this. I initially removed `do not install`

Reply:
done